main.go

```diff
@@ -12,7 +12,7 @@ import (
 func main() {
 	var displaytype = "simple"
 	if len(os.Args) < 2 {
-		fmt.Println("No URI provided !")
+		fmt.Println("Usage: innotopgo mysql://<username>:<password>@<host>:3306")
```

Comment:
if 3306 is used, then the port is not mandatory.
I would suggest to have something like : 

`   mysql://<username>[:<password>]@<host>[:<port>]`

Reply:
Yes, I agree on the port text change, and have just tested it (to verify myself).
But the password is now not mandatory?  Is that assuming a bad practice blank default MySQL password?!

Reply:
You are right, I was confused with MySQL Shell integration of the credential-store-helper  (I thought the driver would use it).
But it's indeed not the case.